commons/src/methods/convertDateTimeZone.ts

```diff
@@ -0,0 +1,46 @@
+import { DateTime } from "luxon";
```

Comment:
Optional, but I want to standardize/componentize date and time strings, is it possible if you could use/make a function in the datetime file to convert to string instead? 

This way if we ever want to change how the string looks it will propagate across all components + devs don't need to think about how the string will look, just need to grab the fn.

Reply:
I can't pass `dateTimeSlot` to `getTimeString()`, but I can change two of these to use `getTimeString`